controllers/articles.controller.js

```diff
@@ -0,0 +1,19 @@
+const { lock } = require("../app");
+const selectArticleById = require("../models/articles.model");
+
+const getArticlesBeId = (req, res, next) => {
+	const { article_id } = req.params;
+
+	selectArticleById(article_id)
+		.then((article) => {
+			res
+				.status(200)
+				.send({ article })
+				.catch((err) => next(err));
+		})
+		.catch((err) => {
```

Comment:
We should only have one catch block in a promise chain and this is the one i would keep